docset/windows/vamt/find-vamtmanagedmachine.md

```diff
@@ -69,6 +69,14 @@ PS C:\>Find-VamtManagedMachine -QueryType ldap -QueryValue "ldap://domainname/??
 This command searches for computers by using an LDAP query.
 The search results are restricted to the computers that have LabComp in their name.
 
+### Example 5: Search in a domain and store results in specified database
+```
+PS C:\>Find-VamtManagedMachine -QueryType ActiveDirectory -QueryValue "DomainName" -MachineFilter "labcomp*" -DbConnectionString "Data Source=localhost\SQLEXPRESS;Initial Catalog=VAMT;Integrated Security=True;MultipleActiveResultSets=True"
+```
+
+This command searches the specified Active Directory domain.
+The search results are restricted to the computers that have LabComp in their name and the results are stored in Local SQLEXPRESS instance.
```

Comment:
```suggestion
The search results are restricted to computers that have LabComp in their name, and the results are stored in Local SQLEXPRESS instance.
```

Reply:
The first part here is copied again from Example 2 so it should be corrected in both places.

Reply:
Up to you, we don't see the context, so please use your judgement. Thanks.

Reply:
I have fixed the comma. For the other suggestion as it is the same in all examples, I prefer not to change it and let the auther decided if change is needed.
Thanks.

Reply:
Thanks, appreciate your thoughtful application of review suggestions.